api/src/services/config-watcher.js

```diff
@@ -143,14 +144,19 @@ const updateServiceWorker = () => {
   });
 };
 
-const load = () => {
+const addSystemUserToDbs = () => Promise.all([...DATABASES, { name: `${environment.db}-purged-cache` }]
```

Comment:
Technically this is not currently "config related", so it probably does not belong in here.  I just stuffed it in here for now since it was the right place in the startup and nowhere else seemed better.

Also probably need a more centralized approach to listing all the DBs that the system user should have member access too....